Add back version.json for dockerflow

whd opened the pull request:
Fallout from https://github.com/mozilla/missioncontrol/pull/39/, we need this file to rebuild dev for the celery changes.

wlach commented:
Sorry about breaking this. I later found out there is a more universally accepted way to not send all the things to the docker container -- using a .dockerignore file. @whd - do you want doing that instead? That approach hopefully won't break local development.

Example:

https://github.com/mozilla/telemetry-batch-view/pull/267

whd commented:
@wlack I updated the PR to ignore version.json and added a touch to the build script to avoid breaking local environments. However, .dockerignore sounds like the right way to go.

whd commented:
@wlach I'm going to close this, please file a PR with the recursive copy and a .dockerignore file since I'm not sure which files you wanted to exclude.
